internal/resolvers/youtube/initialize_test.go

```diff
@@ -18,6 +19,11 @@ func TestInitialize(t *testing.T) {
 	pool, err := pgxmock.NewPool()
 	c.Assert(err, qt.IsNil)
 
+	// google.golang.org/api/youtube/v3 automatically
+	// uses this environment variable to initialize clients
+	err = os.Unsetenv("GOOGLE_APPLICATION_CREDENTIALS")
```

Comment:
You should be able to use `c.Unsetenv` instead, making sure it's recovered by the process after this test is run. Not very important for this specific environment variable, but might as well